api/krusty/kustomizer.go

```diff
@@ -112,10 +112,16 @@ func (b *Kustomizer) Run(
 	}
 	m.RemoveBuildAnnotations()
 	if !utils.StringSliceContains(kt.Kustomization().BuildMetadata, types.OriginAnnotations) {
-		m.RemoveOriginAnnotations()
+		err = m.RemoveOriginAnnotations()
+		if err != nil {
+			return nil, err
```

Comment:
```suggestion
			return nil, errors.WrapPrefixf(err, "failed to clean up origin tracking annotations")
```

and similar below (you'll need to add `"sigs.k8s.io/kustomize/kyaml/errors"` import to the file as well)

Reply:
done [4e7f4bc](https://github.com/kubernetes-sigs/kustomize/pull/4663/commits/4e7f4bce7be27cb022711c41717a8410ea5ee773)